Plots/Contours/NCL_eof_1_1.py

```diff
@@ -0,0 +1,320 @@
+
+print('\n\neof_ts:\n\n')
+print(eof_ts)
+
+###############################################################################
+# Normalize time series:
+
+# Sum spatial weights over the area used.
+nLon = len(xw['lon'])
+clat_subset = clat.where((clat.lat >= latS) & (clat.lat <= latN), drop=True)
+weightTotal = clat_subset.sum() * nLon
+eof_ts = eof_ts / weightTotal
+
+print('\n\neof_ts normalized:\n\n')
+print(eof_ts)
+
+###############################################################################
+# Utility function:
+
+# Define a utility function for creating a contour plot.
+def make_contour_plot(ax, dataset):
+
+    lat = dataset['lat']
+    lon = dataset['lon']
+    values = dataset.data
+
+    # Import an NCL colormap
+    cmap = gvcmaps.BlWhRe
+
+    # Specify contour levels
+    v = np.linspace(-0.08, 0.08, 9, endpoint=True)
+
+    # The function contourf() produces fill colors, and contour() calculates contour label locations.
+    cplot = ax.contourf(lon, lat, values, levels=v, cmap=cmap, extend="both")
+    p = ax.contour(lon, lat, values, levels=v, linewidths=0.0)
+
+    # Label the contours
+    ax.clabel(p, fontsize=8, fmt="%0.2f", colors="k")
+
+    # Add coastlines
+    ax.coastlines(linewidth=0.5)
+
+    # Use geocat.viz.util convenience function to add minor and major tick lines
+    gvutil.add_major_minor_ticks(ax, x_minor_per_major=3, y_minor_per_major=4, labelsize=10)
+
+    # Use geocat.viz.util convenience function to set axes tick values
+    gvutil.set_axes_limits_and_ticks(ax, xticks=[-60, -30, 0, 30], yticks=[40, 60, 80])
+
+    # Use geocat.viz.util convenience function to make plots look like NCL plots, using latitude & longitude tick labels
+    gvutil.add_lat_lon_ticklabels(ax)
+
+    return cplot, ax
+
+
+###############################################################################
+# Plot (1): Draw a contour plot for each EOF
+
+# Generate figure and axes using Cartopy projection  and set figure size (width, height) in inches
+fig, axs = plt.subplots(neof, 1, constrained_layout=True, subplot_kw={"projection": ccrs.PlateCarree()}, figsize=(6, 9))
+
+# Add multiple axes to the figure as contour and contourf plots
+for i in range(neof):
+
+    eof_single = eof.sel(evn=i)
+
+    # Create contour plot for the current axes
+    cplot, axs[i] = make_contour_plot(axs[i], eof_single)
+
+    # Use geocat.viz.util convenience function to add titles to left and right of the plot axis.
+    pct = eof.pcvar[i]
+    gvutil.set_titles_and_labels(axs[i], lefttitle=f'EOF {i + 1}', lefttitlefontsize=10,
+                                 righttitle=f'{pct:.1f}%', righttitlefontsize=10)
+
+# Add horizontal colorbar
+cbar = plt.colorbar(cplot, ax=axs, orientation='horizontal', shrink=0.6)
+cbar.ax.tick_params(labelsize=8)
+
+# Set a common title
+axs[0].set_title(f'SLP: DJF: {yearStart}-{yearEnd}', fontsize=14, y=1.12)
+
+# plt.savefig('test.png')
+plt.savefig('eof_1_1_lg.png')
+
+# Show the plot
```

Comment:
Remove?

Reply:
Yes, thanks a lot for catching this! Removed it with a new commit.